data/eos-autoupdater.conf

```diff
@@ -10,4 +10,5 @@
 #  3 - apply updates
 LastAutomaticStep=3
 IntervalDays=14
+RandomizedDelayDays=7
```

Comment:
Done. I’ve filed a PR for the configuration change as https://github.com/endlessm/eos-image-builder/pull/296.